docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -61,6 +61,9 @@ service:
 ```
 
 ## OTLP
+{{% admonition type="note" %}} 
+If you are on Grafana Cloud, please use the OpenTelemetry Protocol (OTLP), and you can find more information [here] (https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
```

Comment:
I don't think it makes sense to say to use OTLP, since this section is in fact about using OTLP. What we want to instead is to recommend using the OTel Gateway endpoint in Grafana Cloud.

Reply:
As @aknuds1 has pointed out, it is important to write useful link text for those reading the documentation on a screen reader.

For guidance, refer to [Useful link text](https://grafana.com/docs/writers-toolkit/write/links/useful-link-text/)

> I don't think it makes sense to say to use OTLP, since this section is in fact about using OTLP. What we want to instead is to recommend using the OTel Gateway endpoint in Grafana Cloud.

I agree, perhaps the following, also applied to the other note.

```suggestion
To send OTLP data to Grafana Cloud, refer to [Send data using OpenTelemetry Protocol (OTLP)](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
```

I also think the note makes sense at the beginning of this page rather than under the OTLP heading.

Reply:
I actually like @jdbaldry's suggestion better than my own actually, especially since it doesn't mention OTel Gateway (which may be replaced in the future).

Reply:
> do you think it's problematic at all to include Grafana Cloud specifics in Mimir OSS docs?

That's a great question that I'm not certain of the answer to. I will find out and get back to you.

Reply:
Another interesting question.

I am not certain about the intention of this page but at a scan, I believed it to be a task intending to help the user write OpenTelemetry metrics to Mimir. In that case I think it makes sense to tell them as early as possible in the process.

I'm not sure if I would expect users to be linked directly to the OTLP heading unless they are certainly interested in sending to OSS Mimir.

Reply:
@jdbaldry ah my bad, I was confusing this change and the other one. The latter is the HTTP API reference page, and is for a lot more than just OTel.

Reply:
No worries at all, I think for HTTP API you are absolutely right. I'd also consider leaving out the note in that page because I think it is less relevant there. I'm on the fence to be honest. Let's see what the rest of the doc squad has to say about Grafana Cloud notes in OSS docs first :)

Reply:
Got some feedback on that actually, it's fine to have Grafana Cloud notes in the docs, especially if it eliminates confusion.